uast/semantic-uast.md

```diff
@@ -56,6 +56,8 @@ All other field are defined by the Semantic UAST schema.
 
 ### Types
 
+Types are defined in [the SDK](`https://github.com/bblfsh/sdk/blob/87d72062a52cbfe5b52a606f98ec53a29ac94e28/uast/uast.go`). In case of doubt use that source file as reference.
```

Comment:
Was it intentional to link to a specific Git blob instead of a branch?

Reply:
Fixed!